src/pipert2/utils/method_data.py

```diff
@@ -8,5 +8,7 @@
 
 @dataclass
 class Method:
-    name: str
+    event_name: str
+    flow_name: str = None
+    routine_name: str = None
```

Comment:
Maybe change from 2 strings to a dictionary containing the flow_name as a key and list of routine_names as value.
That way you can execute certain event on multiple flows and routines in one call.